CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 1.7.2 (2023-11-27)
+
+ENHANCEMENT:
+* Update Axios to 1.6.2 by @xiehan in https://github.com/hashicorp/js-releases/pull/206
```

Comment:
I assume this was generated but I'm not entirely sure it needs to be surfaced as _enhancement_ unless we're convinced that it is enhancing something inside of `js-releases`.